vital/api/sleep.py

```diff
@@ -38,7 +38,7 @@ def get_sleep_with_stream(
         GET Sleep data with stream data.
         """
         return self.client.get(
-            f"/sleep/{user_id}/stream",
+            f"/summary/sleep/{user_id}/stream",
```

Comment:
Wait this is incorrect

Reply:
No, the url is under the summary endpoints